examples/example_utils.c

```diff
@@ -0,0 +1,23 @@
+#include "example_utils.h"
+
+
+void displayFilterState(const mip_filter_mode filter_state, char **current_state, bool isFiveSeries) 
+{
+    char *read_state = "";
+    if (filter_state == MIP_FILTER_MODE_INIT) {
```

Comment:
This might be better as a switch statement. Also new line the curly braces or remove them entirely. Most of the MIP SDK doesn't use braces for simple statements

Reply:
Removed the curly braces. I'm getting an error when converting the filter state if statements to a switch statement: "expression must have a constant value". Any idea how to get around this? The filter modes are constants.